roles/nat64_appliance/files/elements/nat64-router/install.d/70-dnsmasq

```diff
@@ -21,7 +21,7 @@ set -o pipefail
 
 pushd /root
 
-git clone git://thekelleys.org.uk/dnsmasq.git
+git clone --depth 1 --branch v2.89 git://thekelleys.org.uk/dnsmasq.git
```

Comment:
Guess we can't get a parameter there? static files and all?

Reply:
I think we can, but it would require some work. Pass a variable to diskimage-builder. I'd prefer to do it in follow-up.

Reply:
I added a TODO comment here.